src/Home/Home/Survey/components/Attributes.tsx

```diff
@@ -41,6 +43,9 @@ const Attributes: FC<Props> = ({ occ, sample, isDefaultSurvey }) => {
       <span className={`stage ${stageLocked ? 'locked' : ''}`}>
         <T>{stage}</T>
       </span>
+      <span className={`type ${typeLocked ? 'locked' : ''}`}>
+        <T>{type}</T>
+      </span>
 
```

Comment:
Having added in the `type` field; I added this here to display it along with the other attributes - I _think_ it's for the readonly view